documentation/tut/toy-example-from-scratch.rst

```diff
@@ -397,3 +403,7 @@ Now, we'll reschedule the battery while taking into account the solar production
         --soc-at-start 50% --roundtrip-efficiency 90%
     New schedule is stored.
 
+We can see the updated scheduling in the `FlexMeasures UI <http://localhost:5000/sensors/1/>`_ :
+
+.. image:: https://github.com/FlexMeasures/screenshots/raw/main/tut/toy-schedule/sensor-data-charging-with-solar.png
+    :align: center
```

Comment:
Can you add an image for the asset page, that now include the solar data? I'd like to show the solar data on top of the battery schedule, so the battery should have its `"sensors_to_show"` attribute set to `[2, [3, 1]]`. I believe that's already the case. Otherwise, you may need to edit this asset attribute in the UI.

As before, please include an introductory sentence, like:

```
The `asset page for the battery <http://localhost:5000/assets/1/>`_ now shows the solar data, too.
```